meson.build

```diff
@@ -0,0 +1,43 @@
+project(
+    'GTKChess',
+    'c',
+    version: '0.0.1',
+    license: 'WTFPL',
+    default_options: [
+        'c_std=c18',
+        'warning_level=2',
+        'optimization=3'
+    ]
+)
+
+add_project_arguments('-Wno-overlength-strings', language: 'c')
+
+dep_gtk3 = dependency('gtk+-3.0', version: '>= 3.24.0')
+dep_glib = dependency('glib-2.0', version: '>= 2.64.0')
+dep_giounix = dependency('gio-unix-2.0', version: '>= 2.64.0')
+dep_rsvg = dependency('librsvg-2.0', version: '>= 2.55.0')
+
+subdir('data')
+
+gnome = import('gnome')
+gresources = gnome.compile_resources(
+    'UI',
+    gresources_file,
+    source_dir: 'data'
+)
+
+subdir('src')
+executable(
+    'engine_manager',
+    engine_manager_sources,
+    dependencies: [dep_giounix],
+    link_args: ['-pthread']
```

Comment:
This is probably better done with `dependency('threads')`.

Reply:
It definitely is, thanks!